directory/tests/conftest.py

```diff
@@ -13,14 +13,19 @@
 
 
 @pytest.fixture(scope="session")
-def dd_environment():
+def dd_environment(instance):
     compose_file = os.path.join(common.HERE, "compose", "docker-compose.yml")
     with docker_run(
         compose_file=compose_file,
     ):
-        yield common.get_config_stubs(".")[0]
+        yield instance
 
 
 @pytest.fixture
 def check():
     return DirectoryCheck(common.CHECK_NAME)
+
+
+@pytest.fixture(scope='session')
```

Comment:
I do no recommend this. We want to get a new copy of the instance for every test in case we want to do some modification to it in the test without affecting others.
Setting the `session` scope will prevent the instance from being recreated every time it is used.